islands/InfoModal.tsx

```diff
@@ -2,12 +2,9 @@ import { ConsumerIcon } from "../components/icons/consumer.tsx";
 import { ProducerIcon } from "../components/icons/producer.tsx";
 import IconPlus from "https://deno.land/x/tabler_icons_tsx@0.0.3/tsx/plus.tsx";
 import IconX from "https://deno.land/x/tabler_icons_tsx@0.0.3/tsx/x.tsx";
-import { attachPipeline } from "../lib/fetch.ts";
-import {
-  PipelineInfo,
-  ServiceInfo,
-} from "https://deno.land/x/snitch_protos@v0.0.56/protos/info.ts";
+
```

Comment:
I probably broke stuff in here. Just tried to make ram in the new data shape without testing too much. 😬